spark/dl/src/main/scala/com/intel/analytics/bigdl/example/languagemodel/PTBWordLM.scala

```diff
@@ -66,15 +66,20 @@ object PTBWordLM {
 
       val model = if (param.modelSnapshot.isDefined) {
         Module.loadModule[Float](param.modelSnapshot.get)
+      } else if (param.withTransformerModel) {
+        PTBModel.transformer(
+          inputSize = param.vocabSize,
+          hiddenSize = param.hiddenSize,
+          outputSize = param.vocabSize,
+          numLayers = param.numLayers,
+          keepProb = param.keepProb)
       } else {
-        val curModel = PTBModel(
+        PTBModel(
```

Comment:
PTBModel.lstm ? we need to keep consistent in terms of  different model naming

Reply:
ok, changed